tests/system/actions.go

```diff
@@ -13,13 +14,24 @@ type akResult struct {
 	returnCode int
 }
 
+func splitToArgs(cmdArgs string) []string {
```

Comment:
What happens if we have a runaway quote (`a "b c" "d e`), it this an error, if not - how do we handle it?

Reply:
I'm not trying to build generic generic args splitter.  So in runaway quote it will to extract expected arguments and thus will will fail specific system test

Reply:
OK